wgpu-hal/src/vulkan/device.rs

```diff
@@ -668,6 +671,20 @@ impl super::Device {
             view_formats.push(desc.format)
         }
 
+        if desc.usage.contains(crate::TextureUses::EXTERNAL) {
+            wgt::strict_assert!(
+                external_queue_family_index.is_none(),
+                "Texture has TextureUse::EXTERNAL, but does not specify the owning queue family"
+            );
+        }
+
+        if external_queue_family_index.is_none() {
+            wgt::strict_assert!(
+                desc.usage.contains(crate::TextureUses::EXTERNAL),
+                "Texture specifies external queue family ownership but does not have TextureUse::EXTERNAL"
+            );
+        }
```

Comment:
Aren't these two assertions wrong ? They don't match the error message or the comments above